core/snippets/snippets/markdown.snippet

```diff
@@ -6,3 +6,40 @@ phrase: link
 -
 [$1]($0)
 ---
+
+name: image
+phrase: image
+-
+![$1]($0)
+---
+
+
+name: checkbox
+phrase: to do | check box | tick box
+-
+* [ ] $0
+---
+
+name: bold
```

Comment:
Markdown syntax is not defined as bold/italic/bold italic but just as increasingly strong forms of emphasis; perhaps something like emph[asis]/strong/very strong? Even newer definitions like GitHub Flavored Markdown don't include any definitions of how the various emphasis formats should be rendered (see https://github.github.com/gfm/#emphasis-and-strong-emphasis)

Reply:
Hmm, that's an interesting note. Because intuitively, and for the purpose of saying things, I feel like most people would still refer to it as bold/italic, since that is the most common rendering fashioned by far. However, it is technically incorrect… Perhaps we just include both as options?

Reply:
I'm OK with that if the rendering is so consistent across different Markdown flavors/renderers.

Reply:
Added both in case anyone uses the other as a phrase. Other than that, ready to merge?